Author: Andreas Beckmann <anbe@debian.org>
Description: fix compilation for Linux 4.5
 adjust for this upstream rename:
 net: Rename NETIF_F_ALL_CSUM to NETIF_F_CSUM_MASK
 https://github.com/torvalds/linux/commit/a188222b6ed29404ac2d4232d35d1fe0e77af370

--- a/src/r8168_n.c
+++ b/src/r8168_n.c
@@ -4209,7 +4209,11 @@ static netdev_features_t rtl8168_fix_fea
         spin_lock_irqsave(&tp->lock, flags);
         if (dev->mtu > ETH_DATA_LEN) {
                 features &= ~NETIF_F_ALL_TSO;
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4,5,0)
                 features &= ~NETIF_F_ALL_CSUM;
+#else
+                features &= ~NETIF_F_CSUM_MASK;
+#endif
         }
         spin_unlock_irqrestore(&tp->lock, flags);
 
